core/lib/scmi-server/sub.mk

```diff
@@ -167,6 +167,7 @@ endif
 endif
 
 cflags-lib-$(CFG_SCPFW_MOD_$4) += -DBUILD_HAS_MOD_$4
+scpfw-cmake-flags-y += -DCFG_SCPFW_MOD_$4=$(CFG_SCPFW_MOD_$4)
```

Comment:
@vingu-linaro: considering your P-R #6769, the change above would deserve to factorize declaration of CMake directives in SCP-firmware as part of your [M-R #948](https://gitlab.arm.com/firmware/SCP-firmware/-/merge_requests/948): patch for _SCP-firmware/arch/none/optee/CMakeLists.txt_ (or maybe to apply in _SCP-firmware/product/optee/common/CMakeList.txt_):
[0001-product-optee-declare-CFG_SCPFW_MOD_xxx-CMake-direct.patch.txt](https://github.com/OP-TEE/optee_os/files/14995422/0001-product-optee-declare-CFG_SCPFW_MOD_xxx-CMake-direct.patch.txt)

Reply:
@etienne-lms:  Yes, I have added it into my SCP-firmware MR. I will push this new MR soon and will put your 2 other patches on top of this PR

Reply:
Thanks.